JMMServer/Providers/MyAnimeList/MALHelper.cs

```diff
@@ -530,15 +532,21 @@ public static void UpdateMALSeries(AnimeSeries ser)
                             ser.GetAnime().AnimeID);
                         continue;
                     }
+
+                    string confirmationMessage = "";
+                    if (animeListHashtable.ContainsKey(malID))
```

Comment:
> I don't think there is an existing condition, I am just pointing out that in case of mass update this API call will most likely give you outdated list thus it can fail in case where you delete series but MAL still says it is there, so it will try to update instead of add and will fail.

That's definitely a good point. Due to a lack of good ways to control using Add or Modify, we could always fall back to calling both commands one after the other. While I think it's wasteful to do it that way, I still think it's better than having un-updateable series in MAL. Thanks for taking the time to look at this though - criticism is welcome and I just want to see all these series that were stuck without watch progress / votes in MAL get updated again. 

By the way, there actually appears to be some disturbing behavior regarding Update. I was testing the case you're talking about by attempting to use MAL API Update on a series that is not in my MAL. I ran an Update call through curl and updated series 2168 with 1 episode and watched status. MAL API responded that this was SUCCESSFUL, it even added it to my anime history
"Sakura Taisen: New York ep. 1"
but did not add it to my list / change the watched status. 

So in the case where we attempt to update a series that is no longer / never been in our MAL, it will do nothing and create a phantom history for the command. I have not encountered this personally except in this particular test case, but you're right in that it could happen. However, due to MAL's lack of useful responses (201 Created / Updated being the only things they respond with), I am not sure what to do.